src/bilder/images/edxapp_v2/templates/edxapp/mitxonline/cms_only.yml.tmpl

```diff
@@ -17,3 +17,13 @@ SOCIAL_AUTH_EDX_OAUTH2_SECRET: {{ .Data.studio_oauth_client.secret }}
 SOCIAL_AUTH_EDX_OAUTH2_URL_ROOT: https://{{ key "edxapp/lms-domain" }}  # possibly same as public LMS root URL
 SOCIAL_AUTH_EDX_OAUTH2_PUBLIC_URL_ROOT: https://{{ key "edxapp/lms-domain" }}
 SESSION_COOKIE_NAME: {{ env "ENVIRONMENT" }}-edx-studio-sessionid  # MODIFIED
+CALCULATOR_HELP_URL: ''
```

Comment:
This looks good, but it would be helpful to group these under a comment block explaining their purpose.

Reply:
Added in https://github.com/mitodl/ol-infrastructure/pull/2138/commits/fe3f9f97b67c69a7bad78e56867043d89ebca39b